File: CalculatorProject/src/input_output.py
class InputOutput:
    def __init__(self):
        pass

    def display_menu(self):
        print("Calculator Menu:")
        print("1. add")
        print("2. subtract")
        print("3. multiply")
        print("4. divide")
        print("Type 'exit' to quit.")

    def get_operation(self):
        return input("Choose an operation: ")

    def get_number(self, prompt):
        return float(input(prompt))

    def display_result(self, result):
        print("Result:", result)

    def display_error(self, error_message):
        print("Error:", error_message)

File: CalculatorProject/src/InputOutput.java
import java.util.Scanner;

public class InputOutput {
    private Scanner scanner = new Scanner(System.in);

    public void displayMenu() {
        System.out.println("Calculator Menu:");
        System.out.println("1. add");
        System.out.println("2. subtract");
        System.out.println("3. multiply");
        System.out.println("4. divide");
        System.out.println("Type 'exit' to quit.");
    }

    public String getOperation() {
        System.out.print("Choose an operation: ");
        return scanner.nextLine();
    }

    public double getNumber(String prompt) {
        System.out.print(prompt);
        return Double.parseDouble(scanner.nextLine());
    }

    public void displayResult(double result) {
        System.out.println("Result: " + result);
    }

    public void displayError(String errorMessage) {
        System.out.println("Error: " + errorMessage);
    }
}
